arrow-optics/src/test/kotlin/arrow/optics/GetterTest.kt

```diff
@@ -140,43 +129,43 @@ class GetterTest : UnitSpec() {
       }
     }
 
-    "Asking for the focus in a Reader" {
-      forAll(genToken) { token: Token ->
-        tokenGetter.ask().runId(token) == token.value
-      }
-    }
-
-    "toReader is an alias for ask" {
-      forAll(genToken) { token: Token ->
-        tokenGetter.ask().runId(token) == tokenLens.toReader().runId(token)
-      }
-    }
-
-    "Asks with f is the same as applying f to the focus of the lens" {
-      forAll(genToken, Gen.functionAToB<String, String>(Gen.string())) { token, f ->
-        tokenGetter.asks(f).runId(token) == f(token.value)
-      }
-    }
-
-    "Extract should extract the focus from the state" {
-      forAll(genToken) { generatedToken ->
-        tokenGetter.extract().run(generatedToken) ==
-          State { token: Token ->
-            token toT tokenGetter.get(token)
-          }.run(generatedToken)
-      }
-    }
-
-    "toState should be an alias to extract" {
-      forAll(genToken) { token ->
-        tokenGetter.toState().run(token) == tokenGetter.extract().run(token)
-      }
-    }
-
-    "extractMap with f should be same as extract and map" {
-      forAll(genToken, Gen.functionAToB<String, String>(Gen.string())) { token, f ->
-        tokenGetter.extractMap(f).run(token) == tokenGetter.extract().map(f).run(token)
-      }
-    }
+//    "Asking for the focus in a Reader" {
+//      forAll(genToken) { token: Token ->
+//        tokenGetter.ask().runId(token) == token.value
+//      }
+//    }
+//
+//    "toReader is an alias for ask" {
+//      forAll(genToken) { token: Token ->
+//        tokenGetter.ask().runId(token) == tokenLens.toReader().runId(token)
+//      }
+//    }
+//
+//    "Asks with f is the same as applying f to the focus of the lens" {
+//      forAll(genToken, Gen.functionAToB<String, String>(Gen.string())) { token, f ->
+//        tokenGetter.asks(f).runId(token) == f(token.value)
+//      }
+//    }
+//
+//    "Extract should extract the focus from the state" {
+//      forAll(genToken) { generatedToken ->
+//        tokenGetter.extract().run(generatedToken) ==
+//          State { token: Token ->
+//            token toT tokenGetter.get(token)
+//          }.run(generatedToken)
+//      }
+//    }
+//
+//    "toState should be an alias to extract" {
+//      forAll(genToken) { token ->
+//        tokenGetter.toState().run(token) == tokenGetter.extract().run(token)
+//      }
+//    }
+//
+//    "extractMap with f should be same as extract and map" {
+//      forAll(genToken, Gen.functionAToB<String, String>(Gen.string())) { token, f ->
+//        tokenGetter.extractMap(f).run(token) == tokenGetter.extract().map(f).run(token)
+//      }
+//    }
```

Comment:
These can now be removed from here.

```suggestion
```

Reply:
yes!! thank you simon! Tests were moved.. but forgot to remove the comments! 👍